model/SupplyChain/SupplyChain.md

```diff
@@ -0,0 +1,16 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SupplyChain
+
+## Summary
+
+The supply chain profile documents events and processes associated with the lifecyle of a product, including its creation, transportation, usage, and decommissioning.
```

Comment:
```suggestion
The SupplyChain profile documents events and processes associated with the lifecyle of a product, including its creation, transportation, usage, and decommissioning.
```

Not entirely sure about this, but I think the profile name should be either "SupplyChain" or "Supply Chain", with each word starts with an upper case - as a proper noun.

As a reference, "SimpleLicensing" use Pascal case (upper camel case), without the space between two words:

https://github.com/spdx/spdx-3-model/blob/12174d04bc8e591b5674700ff4ac72b85e4f9d41/model/SimpleLicensing/SimpleLicensing.md?plain=1#L11

@RexJaeschke @zvr @goneall @kestewart what do you think?